cargo-afl/src/main.rs

```diff
@@ -308,11 +308,12 @@ where
         }
 
         rustflags.push_str(&format!(
-            "-Z llvm-plugins={p}/cmplog-instructions-pass.so  \
-            -Z llvm-plugins={p}/cmplog-routines-pass.so \
+            "-Z llvm-plugins={p}/afl-llvm-dict2file.so \
             -Z llvm-plugins={p}/cmplog-switches-pass.so \
+            -Z llvm-plugins={p}/split-switches-pass.so \
             -Z llvm-plugins={p}/SanitizerCoveragePCGUARD.so \
```

Comment:
Sorry if this is a dumb question, but how did you choose where in the order to put this plugin (`SanitizerCoveragePCGUARD.so`)?

Reply:
https://github.com/AFLplusplus/AFLplusplus/blob/stable/src/afl-cc.c#L3482-L3484

Reply:
the order is the correct one. first cmplog should know about the switch compare, then we split the switches so the coverage can easily see the differences in comparing the switches - then the coverage instrumentation. later the instrumentations of integer/float/string/pointer comparisons.